docs/source/stats.rst

```diff
@@ -210,6 +210,13 @@ to one way ANOVA, but still in developement
 
    pairwise_tukeyhsd
 
+.. currentmuodule:: statsmodels.stats.multitest
+
+.. autosummary::
+   :toctree: generated/
+
+   local_fdr
+
```

Comment:
And `fdrcorrection_twostage` maybe? `fdrcorrection` is in the above toctree as `fdrcorrection0`, but I don't see two stage.